tests/foreman/api/test_contentview.py

```diff
@@ -507,6 +523,53 @@ def test_negative_create_with_invalid_name(self):
                 with self.assertRaises(HTTPError):
                     entities.ContentView(name=name).create()
 
+    @tier2
+    def test_composite_content_view_with_same_repos(self):
+        """Create a Composite Content View with content view having same yum repos.
+        Add filters on the content views, package count for composite should not
+        be changed.
+
+        :id: 957f3758-ca1e-4a1f-8e7d-171750e0eb87
+
+        :expectedresults: package count for composite should not be changed in case of mismatch.
+
+        :bz: 1639390
+
+        :CaseImportance: Medium
+        """
+        org = entities.Organization().create()
+        product = entities.Product(organization=org).create()
+        repo = entities.Repository(
+            content_type='yum',
+            product=product,
+            url=CUSTOM_MODULE_STREAM_REPO_2).create()
+        repo.sync()
+        content_view_1 = entities.ContentView(organization=org).create()
+        content_view_2 = entities.ContentView(organization=org).create()
+
+        # create content views with same repo and different filters
+        for content_view, package in [(content_view_1, 'camel'), (content_view_2, 'cow')]:
+            content_view.repository = [repo]
+            content_view.update(['repository'])
+            content_view__info = self._apply_pacakge_filters(content_view, repo,
```

Comment:
I have query , will this for loop include the below condition as `inclusion=False` for second CV as well?
```
3. create another CV and do the same but with including different RPM
```

Reply:
Yes, it will apply the filters based on the for a loop. 
`for content_view, package in [(content_view_1, 'camel'), (content_view_2, 'cow')]:`

Reply:
As `inclusion=False` so will it including different RPM ?

Reply:
I think you are missing the tick here, `inclusion=False` means we are applying the exclude filters, but the package will be different in both content views. At the first time, it will use the `'camel' `package for content_view_1, and package `'cow'` for content_view_2. 

I think you might be confused because of the method name, I will change it